.github/workflows/danger.yml

```diff
@@ -0,0 +1,26 @@
+name: Danger
+on: [pull_request]
```

Comment:
Does this trigger when they push changes to a branch?

Reply:
Triggers whenever branch is attempting a pull request. Checked upon each event change (such as a push)